nana/modules/animelist.py

```diff
@@ -284,3 +170,46 @@ async def manga_search(client, message):
                 await edrep(message, text=ms_g)
         else:
             await edrep(message, text=ms_g)
+
+
+@app.on_message(filters.user(AdminSettings) & filters.command("favourite", Command))
+async def favourite_animelist(client, message):
+    x = await client.get_inline_bot_results(f"{BotUsername}", f"favourite")
```

Comment:
Codacy found an issue: [f-string is missing placeholders (F541)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330176)